src/content/blog/2025_fosdem_recap.md

```diff
@@ -0,0 +1,134 @@
+This year we had the honor of organizing it, and we worked to put together a Call for Proposal (CfP), and consequently a schedule.
+
+- Schedule: https://fosdem.org/2025/schedule/track/quantum/
+- CfP: https://unitary.foundation/community/FOSDEM_2025/
+
+## What we learned 📚
+
+FOSDEM is massive, and running a devroom is no small task.
+These might be obvious to more seasoned devroom organizers, but we would have benefited from these ahead of time!
+
+- **Collating an awesome lineup** -- Many quantum folks don't have FOSDEM on their radar (yet). If you want a strong lineup, individual solicitations are key to having an exceptional speaker lineup.
+- **Day of logistics** -- Purva (our third co-organizer) couldn't make it to Bruxelles, and two people on-site was barely enough. Managing the video stream, helping the speakers with the projector, moderating Q&A, handling the flow of people entering the room—it was controlled chaos. Big shoutout to [Harshit](https://github.com/TheGupta2012/) for stepping in to help!
+- **Prepare for overflow** -- The room filled up fast, and we had to turn people away. It was sad to turn people around, or kick them out, but necessary to ensure FOSDEM can continue into the future. Hopefully next year we can get a bigger room.
+- **Meet speakers beforehand** -- We met most of the speakers 5 minutes before their talk while they were already standing in front of the room, with a laptop in their hands. A pre-event gathering could warm up the room and foster more community among speakers and attendees alike.
+- **Organizer t-shirts** -- (ok, this one is minor) We picked up our organizer t-shirts right before the devroom start on Sunday, and there were only XL and XXL sizes left.
+
+## What we’ll do differently next time 🎯
+
+Was our organization of the devroom perfect? No!
+Are we going to organize it again? Absolutely yes!
+Next iteration, we want to try some ideas.
+Again, these may not sound like new ideas to more seasoned devroom organizers, but they are things we didn't think of implementing this year.
+
+- **Fix the schedule gaps** -- We scheduled five-minute breaks between talks to ensure we had ample time for transitions, but it ended up feeling more awkward than useful. More back-to-back talks or lightning sessions would be better.
+- **Promote the livestream** -- Sharing a link to watch online earlier on forums and chats, as well as reminding people in our community will allow more people to participate.
+- **Monitor the Element chat** -- In all honesty, we completely forgot to check the chat about the chat until half-way through the schedule. At the end of the day, everything was fine, because there wasn't much engagement there, but here is a note to ourselves to monitor the chat next year.
+- **A Quantum Fringe Event** -- A hackathon, meetup, or social event could allow more people to get involved in quantum, especially since some couldn't make it in the door!
```

Comment:
```suggestion
- **A Quantum Fringe Event** -- A hackathon, meetup, or social event on the Friday or one of the evenings, could allow more people to get involved in quantum, especially since some couldn't make it in the door!
```